pom.xml

```diff
@@ -84,22 +84,22 @@
     <bigquery-connector.version>1.2.1-SNAPSHOT</bigquery-connector.version>
 
     <apache.httpclient.version>4.5.13</apache.httpclient.version>
-    <google.api.grpc.proto-google-iam-v1.version>1.0.5</google.api.grpc.proto-google-iam-v1.version>
+    <google.api.grpc.proto-google-iam-v1.version>1.0.11</google.api.grpc.proto-google-iam-v1.version>
     <google.api-client.version>1.31.3</google.api-client.version>
     <google.api-client-libraries.version>1.31.0</google.api-client-libraries.version>
-    <google.api-bigquery.version>v2-rev20201030-${google.api-client-libraries.version}</google.api-bigquery.version>
+    <google.api-bigquery.version>v2-rev20210303-${google.api-client-libraries.version}</google.api-bigquery.version>
     <google.api-iamcredentials.version>v1-rev20201022-${google.api-client-libraries.version}</google.api-iamcredentials.version>
-    <google.api-storage.version>v1-rev20201112-${google.api-client-libraries.version}</google.api-storage.version>
-    <google.auth.version>0.22.2</google.auth.version>
+    <google.api-storage.version>v1-rev20210127-${google.api-client-libraries.version}</google.api-storage.version>
+    <google.auth.version>0.24.1</google.auth.version>
     <google.auto-value.version>1.7.4</google.auto-value.version>
-    <google.cloud-bigquerystorage.version>1.7.2</google.cloud-bigquerystorage.version>
+    <google.cloud-bigquerystorage.version>1.15.0</google.cloud-bigquerystorage.version>
     <google.flogger.version>0.5.1</google.flogger.version>
```

Comment:
Would you you upgrade flogger's version as well?

https://github.com/google/flogger/releases/tag/flogger-0.6

Reply:
Once `com.google.cloud.bigdataoss:util` depends on newer version of flogger-system-backend, I can remove a workaround to avoid dependency conflict (https://github.com/apache/beam/pull/14578/commits/facd2c039dd1d2890d68ee48762bea271e368ad4#r617759024) from Apache Beam.

Reply:
Done.